extension/chrome/elements/passphrase.ts

```diff
@@ -42,6 +42,9 @@ View.run(class PassphraseView extends View {
     if (!this.orgRules.forbidStoringPassPhrase()) {
       $('.forget').prop('disabled', false);
     }
+    if (this.orgRules.usesKeyManager()) {
+      $('#lost-pass-phrase').text("Ask your IT staff for help if you lost your pass phrase.")
```

Comment:
I think it's not quite neat to have 2 alternative messages in different places ("Lost pass phrase?" in `.htm` file, "Ask your IT..." in `.ts`).
Perhaps, it'd be better to keep them in one place (`.htm`?) and switch divs like some examples below
```
.css('display', 'none');
```

Reply:
Agree, it would be neater to have both alternatives next to one another (in htm), and only switch visibility (in ts). :+1:

Reply:
I have moved the two messages in a div as suggested and uses ts for controlling which message should be shown to the user. I should've learned these coding standards from now on. thanks!